src/frontend/frontend/console.js

```diff
@@ -46,214 +46,113 @@ angular.module('frontend-app')
     };
     var asan_contents = [];
 
-    // contents is an array of lines of address sanitizer output
-    function parse_asan_output(contents) {
-      var traced_main = false;
-      var filepatt = /\/([^\/]+(:[0-9]+|[^\)]+))\)?$/;
-      var addrpatt = /0x[0-9a-f]+/;
-      // 0xff is ... 3 bytes ... left ... 10-byte region
-      var suppallocaddrpatt = /^(0x[0-9a-f]+)(?:[^0-9])*([0-9]+).*(left|right|inside)(?:[^0-9])*([0-9]+)-byte/;
-      // .... 0xff is located in stack ...
-      var suppstackaddrpatt = /(0x[0-9a-f]+).*located in stack.*offset ([0-9]+)/;
-      // 0xff is ... 0 bytes ... left ... global variable 'x' defined in 'foo.c:9:15' (0xff) of size 10
-      var suppglobaladdrpatt = /^(0x[0-9a-f]+)(?:[^0-9])*([0-9]+).*(left|right|inside).*global variable '([^']+)'.*'([^':]+):([0-9]+):([0-9]+)' \((0x[0-9a-f]+)\).*size ([0-9]+)$/;
-      // frame ... 2 object(s)
-      var frameinfopatt = /frame(?:[^0-9])*([0-9]+) object/;
-      // [32, 40) 'x'
-      var framevarpatt = /\[([0-9]+), ([0-9]+)\) '([^']+)'/;
```

Comment:
Well, it may be helpful to know how the stack frame looks.  I don't really have a preference either way here.